runtime/mount/python_lib/lplots/h5/h5ad/process_message.py

```diff
@@ -543,6 +574,69 @@ async def process_h5ad_request(
         finally:
             alignment_is_running = False
 
+    if op == "store_image_transformation":
+        if "image_transformation" not in msg or "id" not in msg:
+            return {
+                "type": "h5",
+                "op": op,
+                "data_type": "h5ad",
+                "key": widget_session_key,
+                "value": {
+                    "error": "`image_transformation` or `id` key missing from message"
+                },
+            }
+
+        image_data = adata.uns["latch_images"][msg["id"]]
```

Comment:
you should def use a get here so this does not explode if it is not found in the backend.

Reply:
Should I use get for "latch_images" as well?